masonry/src/widgets/sized_box.rs

```diff
@@ -337,7 +339,9 @@ impl SizedBox {
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Remove the child widget.
+    ///
+    /// (If this widget has no child, this method does nothing.)
```

Comment:
Should it `debug_panic` instead (again, not really for this PR)?

Reply:
Agreed, not addressing that this PR (I'm not sure it even warrants a TODO comment).